cmd/lotus-shed/fr32.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"io"
+	"os"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/lotus/extern/sector-storage/fr32"
+)
+
+var fr32Cmd = &cli.Command{
+	Name:        "fr32",
+	Description: "fr32 tools",
+	Subcommands: []*cli.Command{
+		fr32WriteCmd,
+		fr32ReadCmd,
+	},
+}
+
+var fr32WriteCmd = &cli.Command{
+	Name:  "write",
```

Comment:
Maybe give it similar semantics as `base64` and other encoding decoding commands.
Encode by default, decode with `-d` flag.
Not sure.